doc/nrf-bm/migration/nrf5_bm_migration.rst

```diff
@@ -255,6 +259,30 @@ This is due to several factors, including:
 * Different project configuration mechanism, inherently affecting how libraries are configured.
 * Different coding standard in the |NCS| (for example, limited use of ``typedef``).
 
+Error codes
+***********
+
+nRF5 mainly used the unsigned nRF error scope for error handling, with errors such as ``NRF_ERROR_NO_MEM``, ``NRF_ERROR_FORBIDDEN`` and ``NRF_SUCCESS`` for successful operations.
+In addition the SoftDevice and nrfx provided BLE and nRFX error codes within the same error scope, but with a diffeent base offset.
```

Comment:
I understand what you want to say here, these were really similar, however they were in effect different errors codes, defined in different files and with different names.